test/e2e/network/no_snat.go

```diff
@@ -154,29 +108,9 @@ var _ = SIGDescribe("NoSNAT [Feature:NoSNAT] [Slow]", func() {
 			framework.ExpectNoError(err)
 		}
 
-		// In some (most?) scenarios, the test harness doesn't run in the same network as the Pods,
```

Comment:
Do we still need to feed the node IP in `newTestPod` or can that be removed now?

Reply:
I think so, so we test the connectivity from all the nodes in the cluster that is being tested, and make the test predictable

Reply:
we can get rid of all the environment variables though

Reply:
> we can get rid of all the environment variables though

yeah was specifically referring to the NODE_IP env var we're adding to all the tests above. Seems enough to just create a test pod per node without the env var now.

Reply:
duh, scratch my comment, you are absolutely right, nodeName is what we need :upside_down_face: